eZ/Publish/Core/Repository/Values/ObjectState/ObjectState.php

```diff
@@ -26,16 +23,33 @@
  */
 class ObjectState extends APIObjectState
 {
-    use MultiLanguageTrait;
-    use MultiLanguageNameTrait;
-    use MultiLanguageDescriptionTrait;
```

Comment:
why `$this->{static::DEFAULT_LANGUAGE_PROPERTY}` instead of `static::DEFAULT_LANGUAGE_PROPERTY` ?